test/notificationSpec.js

```diff
@@ -1013,6 +1014,13 @@ describe('Notifications', function () {
       })
     })
   })
+
+  describe('parseLedger', function () {
```

Comment:
Oops, already merged. I can add that test case, but I wanted to release this and get ILP server working, so that I can finish adding the signing keys to the server by the end of the day.